src/js/components/projects/createProjectModal/index.tsx

```diff
@@ -0,0 +1,482 @@
+
+  const fetchCollaborators = useCallback(
+    async (org: string) => {
+      setCollaborators([]);
+      if (org) {
+        setIsRefreshingCollaborators(true);
+        const response = await apiFetch({
+          url: window.api_urls.organization_members(org),
+          dispatch,
+        });
+        if (response && user?.github_id) {
+          setCollaborators(
+            filter(response, (member) => member.id !== user.github_id),
+          );
+        }
+        setIsRefreshingCollaborators(false);
+      }
+    },
+    [dispatch, user?.github_id],
+  );
+
+  const checkOrgPermissions = useCallback(
+    async (org: string) => {
+      setOrgIsValid(false);
+      setOrgErrors([]);
+      if (org) {
+        // setIsCheckingRepoName(true);
+        const { success, messages }: { success: boolean; messages: string[] } =
+          await apiFetch({
+            url: window.api_urls.organization_check_app_installation(org),
+            dispatch,
+            opts: { method: 'POST' },
+          });
+        setOrgIsValid(success);
+        setOrgErrors(messages);
+      }
+    },
+    [dispatch],
+  );
+
+  // Fetch GitHub Org members and check permissions when organization changes
+  useEffect(() => {
+    if (inputs.organization) {
+      fetchCollaborators(inputs.organization);
+      checkOrgPermissions(inputs.organization);
+    }
+  }, [checkOrgPermissions, fetchCollaborators, inputs.organization]);
+
+  // When available GitHub Org members change, reset selected collaborators
+  const collaboratorsRef = useRef(collaborators);
+  useEffect(() => {
+    const prevValue = collaboratorsRef.current;
+    if (collaborators !== prevValue) {
+      setInputs({
+        ...inputs,
+        github_users: intersectionBy(inputs.github_users, collaborators, 'id'),
+      });
+      collaboratorsRef.current = collaborators;
+    }
+  }, [collaborators, inputs, setInputs]);
+
+  const canSubmit = Boolean(
+    nameIsAvailable &&
+      orgIsValid &&
+      inputs.name &&
+      inputs.organization &&
+      inputs.repo_name,
+  );
+
+  const selectedOrg = inputs.organization
+    ? find(orgs, { id: inputs.organization })
+    : undefined;
+
+  const selectedDeps = filter(dependencies, (d) =>
+    (inputs as CreateProjectData).dependencies.includes(d.id),
+  );
+
+  const doSubmit = (e: React.MouseEvent<HTMLButtonElement>) => {
+    setIsSaving(true);
+    handleSubmit(e as any);
+  };
+
+  const CancelBtn = (args: any) => (
+    <Button
+      label={t('Cancel')}
+      onClick={handleClose}
+      disabled={isSaving}
+      {...args}
+    />
+  );
+
+  const BackBtn = (args: any) => (
+    <Button
+      label={t('Go Back')}
+      variant="outline-brand"
+      onClick={prevPage}
+      {...args}
+    />
+  );
+
+  const NextBtn = (args: any) => (
+    <Button
+      label={t('Save & Next')}
+      variant="brand"
+      onClick={nextPage}
+      {...args}
+    />
+  );
```

Comment:
@dvdherron Any suggestions on how to prevent the text in these buttons from wrapping so quickly?

![Screen Shot 2022-05-24 at 4 20 42 PM](https://user-images.githubusercontent.com/552316/170125224-1e362b2b-d615-4be7-8e69-b474a0cc0938.jpg)

Reply:
@jgerigmeyer Yeah, that works, as long as we are fine with the layout breaking like this on _really_ narrow screens:
![image](https://user-images.githubusercontent.com/41588129/170293797-635ef7fd-e294-4cd0-87ef-1909f9f28e50.png)

For the situation when a user is not a member doesn't have the correct GH Organization permissions, would we be okay with letting the "Re-sync" button wrap so that that input help text can span the page? Otherwise that text gets long in a single column:

wrap:
![image](https://user-images.githubusercontent.com/41588129/170294696-bc2bb557-8872-4983-88b0-1886024b3ee8.png)

no wrap:
![image](https://user-images.githubusercontent.com/41588129/170295640-a8c59b74-a762-4507-aad6-e68f83fdd154.png)

Reply:
Works for me 👍